sass/themes/cr/2019/components/typography/typography.twig

```diff
@@ -26,23 +26,23 @@
   <span class="font--red"><b>.text--title</b> (Used for headers and landing pages, noted as 'Title' H1 in the Design System)</span><br/>
   <span class="breakpoint-lg font--red"><b>Current breakpoint: LG </b> - font-size: 100px / line-height: 104px</span>
   <span class="breakpoint-md font--red"><b>Current breakpoint: MD </b> - font-size: 100px / line-height: 104px</span>
-  <span class="breakpoint-sm font--red"><b>Current breakpoint: SM </b> - font-size: 100px / line-height: 104px</span>
+  <span class="breakpoint-sm font--red"><b>Current breakpoint: SM </b> - font-size: 54px / line-height: 62px</span>
 <br/>
 <br/>
 <br/>
 
 <h1>Header 1</h1>
  <span class="breakpoint-lg font--red"><b>Current breakpoint: LG </b> - font-size: 64px / line-height: 72px</span>
  <span class="breakpoint-md font--red"><b>Current breakpoint: MD </b> - font-size: 64px / line-height: 72px</span>
- <span class="breakpoint-sm font--red"><b>Current breakpoint: SM </b> - font-size: 64px / line-height: 72px</span>
+ <span class="breakpoint-sm font--red"><b>Current breakpoint: SM </b> - font-size: 42px / line-height: 48px</span>
 <br/>
 <br/>
 <br/>
 
 <h2>Header 2</h2>
   <span class="breakpoint-lg font--red"><b>Current breakpoint: LG </b> - font-size: 42px / line-height: 48px</span>
   <span class="breakpoint-md font--red"><b>Current breakpoint: MD </b> - font-size: 42px / line-height: 48px</span>
-  <span class="breakpoint-sm font--red"><b>Current breakpoint: SM </b> - font-size: 42px / line-height: 48px</span>
+  <span class="breakpoint-sm font--red"><b>Current breakpoint: SM </b> - font-size: 32px / line-height: 40px</span>
```

Comment:
Check with font mixin. Is size 32 or 34 px?

Reply:
![clip-art-number-34-clipart-1](https://user-images.githubusercontent.com/4737220/50009328-a2788580-ffae-11e8-80ee-dd8cc8e82d7e.jpg)